_posts/2023-02-22-puppetmodule.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Make puppet module documentation great again!
+date: 2023-02-22
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+A long long time ago I can still remember... [Dominic Cleal](https://github.com/domcleal)
+created [puppetmodule.info](https://www.puppetmodule.info/). Inspired by
+[rubydoc.info](https://rubydoc.info/), the site generates documentation for
+Puppet modules with [puppet-strings](). puppetmodule.info was created around
+2015. Since then Dominic moved on and isn't active anymore in the Puppet
+ecosystem and the site died.
+
+Vox Pupuli adopted it by the end of 2022. We updated the source code to work on
+modern Ruby and enabled TLS for the domain.
+
+You want to use puppetmodule.info for your own modules? Just publish to the
+forge! We scrape the forge API on a regular basis to import new releases
+automatically. You can also add a badge to your README.md:
+
+```
+![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-borg)
```

Comment:
Should we link this to puppet-example? Which does exist as well. Perhaps mostly as my playground, but it's there.

Reply:
I'm fine with every module. borg was the first one that came up to my mind with working puppet-strings docs, but I'm happy to switch to something else.